play/src/front/Components/EmbedScreens/CoWebsitesContainer.svelte

```diff
@@ -1,53 +1,392 @@
 <script lang="ts">
     import { fly } from "svelte/transition";
+    import { onMount } from "svelte";
     import { coWebsites } from "../../Stores/CoWebsiteStore";
-    import XIcon from "../Icons/XIcon.svelte";
     import FullScreenIcon from "../Icons/FullScreenIcon.svelte";
-    import CoWebsiteTab from "./CoWebsiteTab.svelte";
+    import JitsiCowebsiteComponent from "../Cowebsites/JistiCowebsiteComponent.svelte";
+    import SimpleCowebsiteComponent from "../Cowebsites/SimpleCowebsiteComponent.svelte";
     import { JitsiCoWebsite } from "../../WebRtc/CoWebsite/JitsiCoWebsite";
-    import { log } from "console";
-    import { coWebsiteManager } from "../../WebRtc/CoWebsiteManager";
-    import { CoWebsite } from "../../WebRtc/CoWebsite/CoWebsite";
+    import { SimpleCoWebsite } from "../../WebRtc/CoWebsite/SimpleCoWebsite";
+    import { BBBCoWebsite } from "../../WebRtc/CoWebsite/BBBCoWebsite";
+    import BigBlueButtonCowebsiteComponent from "../Cowebsites/BigBlueButtonCowebsiteComponent.svelte";
+    import CoWebsiteTab from "./CoWebsiteTab.svelte";
+    import { max, min } from "lodash";
+    import { ArrowDownIcon } from "svelte-feather-icons";
 
-    export let vertical = false;
-    export let active = false;
-</script>
+    let activeCowebsite = $coWebsites[0].getId();
```

Comment:
Maybe `activeCowebsite` could be the plain Cowebsite object and not the ID of the object.
It will save some code.